aws-iot-mitigationaction/src/main/java/com/amazonaws/iot/mitigationaction/HandlerUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.amazonaws.iot.mitigationaction;
+
+import software.amazon.awssdk.services.iot.IotClient;
+import software.amazon.awssdk.services.iot.model.IotException;
+import software.amazon.awssdk.services.iot.model.ListTagsForResourceRequest;
+import software.amazon.awssdk.services.iot.model.ListTagsForResourceResponse;
+import software.amazon.awssdk.services.iot.model.Tag;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HandlerUtils {
+
+    public static List<software.amazon.awssdk.services.iot.model.Tag> listTags(
```

Comment:
It might be better to pull the common code to a common package, it would be painful for now but reduces the effort for future.

Reply:
Opened an issue about this with the cfn team: https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/issues/335